lib/index.js

```diff
@@ -17,6 +17,12 @@ module.exports = (logger) => {
             err = new Error(err);
         }
 
+        logger.error(`${req.originalMethod}: ${req.url}`);
+
+        if (Object.keys(req.body).length) {
+            logger.error(`Request body: ${JSON.stringify(req.body)}`);
```

Comment:
You can either format the json with 2 spaces for readability
```js
JSON.stringify(req.body, null 2)
```
or let `util.meta` handle the formatting and will depend on the formatting options of the logger say `prettyPrint: true` or `json: true` is set.
```js
logger.error('Request body: ', req.body);
```

Reply:
```javascript
JSON.stringify(req.body, null, 2);
```

Reply:
Hi @FerrielMelarpis, @adin234, thanks for commenting! Right now we don't want to use much of space just to log the req.body, we just need to log the content for easier tracking of error.